linux_pipeline/Jenkinsfile_lis_rpm_hyperv

```diff
@@ -0,0 +1,177 @@
+                                    echo 'Cleaning up LISA environment...'
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+    try{
+        parallel runs
+    } catch (exc){
+        currentBuild.result = 'SUCCESS'
+    }
+}
+
+stage("BVT stage"){
+    def runs = [:]
+    DISTROS = getDistros(DISTRO_VERSIONS, supportedDistros)
+    nodesMap.keySet().each{
+        node = it
+        mappedDistros = nodesMap[it]
+        if (node == 'sriov'){
+            node = 'ws2016'
+        }
+        DISTROS.each{
+            if (mappedDistros.contains("${it},")){
+                runs["${it}-${node}"] = {
+                    node("${node}"){
+                        stage("${it}-${node}"){
+                            withCredentials(bindings: [string(credentialsId: 'AZURE_SAS', variable: 'AZURE_SAS'),
+                                        string(credentialsId: 'LISA_IMAGES_SHARE_URL', variable: 'LISA_IMAGES_SHARE_URL'),
+                                        string(credentialsId: 'LISA_TEST_DEPENDENCIES', variable: 'LISA_TEST_DEPENDENCIES')]) { 
+                                LIS_SUCCESS = false
+                                dir ("b${BUILD_NUMBER}"){
+                                    checkout scm
+                                    try{
+                                        echo 'Running LISA...'
+                                        PowerShellWrapper("""
+                                            & '.\\scripts\\lis_hyperv_platform\\main-lis-rpm.ps1'
+                                                -JobId '${it}-b-${BUILD_NUMBER}'
+                                                -InstanceName '${it}-b-${BUILD_NUMBER}'
+                                                -DistroVersion '${it}' -WorkingDirectory 'C:\\workspace'
+                                                -LISAImagesShareUrl '${LISA_IMAGES_SHARE_URL}' 
+                                                -XmlTest 'lis_pipeline_bvt.xml'
+                                                -LisaTestDependencies '${LISA_TEST_DEPENDENCIES}'
+                                                -LisUrl "\${env:LIS_ARCHIVE_LINK}"
+                                            """)
+                                        LIS_SUCCESS = true
+                                    } finally {
+                                        archiveArtifacts '\\lis-test\\WS2012R2\\lisa\\TestResults\\**\\*'
+                                        junit '\\lis-test\\WS2012R2\\lisa\\TestResults\\**\\*.xml'
+                                    }
+                                }
+                                if (LIS_SUCCESS){
+                                    echo 'Cleaning up LISA environment...'
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+    try{
```

Comment:
do not put everything in to a try catch,
modify the PowerShellWrapper to include returnStatus and return it.
and you can check independently for each job
see https://jenkins.io/doc/pipeline/steps/workflow-durable-task-step/#bat-windows-batch-script